libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/telemetryevents/TelemetryStep.kt

```diff
@@ -0,0 +1,21 @@
+package com.mapbox.navigation.core.telemetry.telemetryevents
+/**
+ * Documentation is here [https://paper.dropbox.com/doc/Navigation-Telemetry-Events-V1--AuUz~~~rEVK7iNB3dQ4_tF97Ag-iid3ZImnt4dsW7Z6zC3Lc]
+ */
+
+// Defaulted values are optional
+
+data class TelemetryStep(
```

Comment:
Same as above `NavigationStepData`

Reply:
TelemetryStep data is required for some of the feedback types. NavigationStepData no longer exists.

Reply:
All Navigation<evnet type>Event have been replaced by Telemetry<Event type>Event